ConfigurationManager/ConfigurationManager.cs

```diff
@@ -207,7 +214,7 @@ string GetCategory(SettingEntryBase eb)
                         .ThenBy(x => x.Key)
                         .Select(x => new PluginSettingsData.PluginSettingsGroupData { Name = x.Key, Settings = x.OrderByDescending(set => set.Order).ThenBy(set => set.DispName).ToList() });
 
-                    return new PluginSettingsData { Info = pluginSettings.Key, Categories = categories.ToList(), Collapsed = settingsAreCollapsed };
+                    return new PluginSettingsData { Info = pluginSettings.Key, Categories = categories.ToList(), Collapsed = collapsedPluginsByName.Contains(pluginSettings.Key.Name) ? !settingsAreCollapsed : settingsAreCollapsed };
```

Comment:
This doesn't seem right, what happens if the collapsed setting is set to off?

Reply:
It's right, but the name was unclear. Changed the name to `nonDefaultCollpasingStateByPluginName`: the entries are added to the set if they do not match the current default. (To allow for new, unknown entries to have the default applied.)